tezos/new_context/src/gc/worker.rs

```diff
@@ -57,15 +56,17 @@ impl Cycles {
         }
 
         let value = value?;
-        self.list
-            .back_mut()
-            .unwrap()
-            .insert(hash_id, Some(Arc::clone(&value)));
+        if let Some(v) = self.list.back_mut() {
+            v.insert(hash_id, Some(Arc::clone(&value)));
+        } else {
+            eprintln!("GC: Failed to insert value in Cycles")
```

Comment:
@bkontur @adonagy do we log output from the protocol runners anywhere? trying to figure out what would be the best way to handle situations like this one (maybe more `Result` is needed here to get these bubbled up, but sometimes a warning is all we need)